manager/src/main/resources/application.properties

```diff
@@ -41,7 +41,7 @@ event-bridge.rhoas.sso.red-hat.refresh-token=empty
 %dev.quarkus.hibernate-orm.log.bind-parameters=true
 
 # SSO
-%dev.quarkus.oidc.auth-server-url=http://localhost:8180/auth/realms/event-bridge-fm
+%dev.quarkus.oidc.auth-server-url=http://192.168.49.2:30007/auth/realms/event-bridge-fm
```

Comment:
Is this value fixed?

Reply:
see https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/221#discussion_r784743494

Reply:
Anyway, I think we might implement a workaround in the near future: somehow we specify in the pom to execute `minikube ip` and we set a property with the result. 
In that way it might be possible to dynamic configure the application for the `dev` profile based on the output of `minikube ip`
https://issues.redhat.com/browse/MGDOBR-227